tools/KoreBuild.Console/Commands/DockerFiles/jessie.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/dotnet:2.0-runtime-deps-jessie
+
+RUN apt-get update && \
+    apt-get install -y git && \
+    apt-get install -y curl && \
+    apt-get install -y unzip && \
+    apt-get install -y apt-transport-https
+
+ADD ./ ./
+
+RUN rm -f ./korebuild-lock.txt
```

Comment:
I think it would be better to exclude this via a `.dockerignore` file, rather than copying and then removing.